### ISSUE TITLE
Neuroimaging experiment

### ISSUE THREAD

**Issue by XinhuiLi** [trendscenter/MISA-pytorch#7]
- [x] Save X and gPCA W from MMIVA project
- [x] Update main.py to save pickle file according to dataset filename and name of W
- [x] Write a slurm job script to iterate through configurations
- [x] Run MATLAB and Python experiments
  - [x] MATLAB simulation
  - [x] Python simulation
  - [x] MATLAB real data
  - [x] Python real data
- [x] Plot configurations vs ISI
- [x] Plot correlations between MATLAB and Python outputs

**Comment by XinhuiLi** [comment]
- [x] Hyper-parameter search
- [ ] Frequency of iterations to get the same MATLAB loss from Python
- [x] ISI: MATLAB/Python configuration (x-sample, y-ISI)
  - [x] Panel: Dataset
  - [x] Panel: Sample
- [x] Loss: Scatter plot of MATLAB/Python loss value (ratio 1)
- [x] Real data visualization: Correlation figure between MATLAB and Python for each modality
- [ ] Spatial map (optional): Nilearn
